packages/hash/frontend/src/components/hooks/useAuthenticatedUser.ts

```diff
@@ -94,6 +94,7 @@ export const useAuthenticatedUser = (
     } else {
       setKratosSession(session);
       setLoadingKratosSession(false);
```

Comment:
Just commenting here because I can't comment elsewhere on the diff. (And again, apologies I'm only thinking about this now rather than on the previous PR), do we even need the `me` query to do what it does?

Presumably we could use `getEntity` and benefit from any patching or recursive queries, etc. that we've defined for it... I guess we'd still need _a_ `me` query or something to get the appropriate `entityId` of the account entity?

Not suggesting we should actually change this but just a thought

Reply:
> do we even need the me query to do what it does?

Interesting, haven't questioned this before. If the alternative is to have a `me` query that returns the `entityId` of the current user, and using the `getEntity` query to get the actual user entity, then we will end up having to do 2 consecutive requests where we previously only had to do one. This might be something worth optimising because the `me` query is fetched on every page-load to determine if the user is logged in, and obtain the latest user entity.

Reply:
That makes sense to me :) I mean ideally 2 requests should be _**very**_ cheap haha, but we've also already got the code so leaving it seems fine.

Reply:
> ideally 2 requests should be very cheap

Server-side these operations are cheap, yes. But because these are browser requests the request time will largely depend on the client's internet connection. Therefore having 2 requests instead of 1 could have a noticeable impact on loading times.